Automattic/sensei
# Add course and share dynamic data

Fixes https://github.com/Automattic/sensei/issues/5633
Based on https://github.com/Automattic/sensei/pull/5933

### Changes proposed in this Pull Request

* It's the final PR for the tasks section.
* It adds the dynamic data to the course illustration in the tasks section.
* It adds the share links to the ready state.
  * Notice that the Instagram share button was replaced with a Tumblr share button.

### Testing instructions


* Publish a course.
* Go to the Sensei Home, and make sure you start seeing the course along with the tasks.
* Complete all the tasks.
* Click on the share links, and make sure it opens a new window to share the course. Notice that if it's public not on the Internet it won't work properly. You can test it by forcing an existing URL in the `coursePermalink` prop of the `Ready` component.


### Screenshot / Video

<img width="1314" alt="Screen Shot 2022-10-18 at 10 51 33" src="https://user-images.githubusercontent.com/876340/196450633-a4f930ba-26a4-4564-902f-13c0ae63bbd8.png">

<img width="1317" alt="Screen Shot 2022-10-18 at 10 50 08" src="https://user-images.githubusercontent.com/876340/196450678-338946c4-ec79-46a8-9a8d-d54058e9d333.png">
